_config/mimetypes.yml

```diff
@@ -756,7 +756,6 @@ SilverStripe\Control\HTTP:
     stl: application/vnd.ms-pki.stl
     str: application/vnd.pg.format
     stw: application/vnd.sun.xml.writer.template
-    sub: image/vnd.dvb.subtitle
```

Comment:
Given that this is for HTTP responses, and the header only supports one type at a time (http://greenbytes.de/tech/webdav/rfc2616.html#rfc.section.14.17), can we just discard these duplicates and forget about it?